app/models/user.rb

```diff
@@ -6,6 +6,7 @@ class UserDisabled < StandardError
 
 class User < ActiveRecord::Base
   rolify
+  has_many :physical_tickets, through: :ticket_purchases, dependent: :destroy
```

Comment:
I tried this but it's not working, it gives a SQL exception because physical ticket does not have conference_id column. So I am looking for a work around for this.:thinking: